files/build_templates/init_cfg.json.j2

```diff
@@ -16,5 +16,21 @@
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
 {% endfor %}
         }
+    },
+    "Feature": {
```

Comment:
Table name should be all caps, as in my example, `"FEATURE": {`

Reply:
Fixed.